### PR TITLE
Add Total Run Cost to Workflow Detail

### PR DESCRIPTION
**Pull request by jmthibault79**
https://github.com/DataBiosphere/firecloud-app/issues/20

Depends on:
https://github.com/broadinstitute/firecloud-orchestration/pull/638
https://github.com/broadinstitute/rawls/pull/930

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/workspace/monitor/common.cljs [hunk 1]
```diff
@@ -5,16 +5,22 @@
...
-   ))
+   [broadfcui.components.foundation-dropdown :as dropdown]))
 
 
 (defn render-date [date]
   (if date
     (str (common/format-date date) " (" (duration/fuzzy-time-from-now-ms (.parse js/Date date) true) ")")
     [:span {:style {:fontStyle "italic"}} "Pending..."]))
 
+(defn render-cost [cost]
```
Comment (jmthibault79) [inline src/cljs/main/broadfcui/page/workspace/monitor/common.cljs:17]: Moved from `submission_details.cljs` - now shared between it and `workflow_details.cljs`

File: src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs [hunk 2]
```diff
@@ -297,7 +297,7 @@
                   [Failures {:data failures}])]])
             (:data props)))]))})
 
-(defn- render-workflow-detail [workflow raw-data workflow-name submission-id bucketName workspace-id gcs-path-prefix]
```
Comment (jmthibault79) [inline src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs:303]: `bucketName` was not used

File: src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs [hunk 3]
```diff
@@ -332,31 +333,45 @@
...
+           cost-response (:cost-response @state)]
        (cond
-         (nil? server-response)
+         (nil? metadata-response)
          (spinner "Loading workflow details...")
-         (not (:success? server-response))
-         (style/create-server-error-message (:response server-response))
+         (not (:success? metadata-response))
+         (style/create-server-error-message (:response metadata-response))
+         (nil? cost-response)
```
Comment (jmthibault79) [inline src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs:346]: Only checks for cost errors after doing so the main metadata errors.  This seems like the right priority to me.

**Review by ghost — COMMENTED** [review]
Looks good 👍

**Pull request merged**